packages/cozy-search/src/components/Search/AssistantLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import AssistantProvider, { useAssistant } from '../AssistantProvider'
+import { isAssistantEnabled, makeConversationId } from '../helpers'
+
+const AssistantLink = ({ children }) => {
+  const { onAssistantExecute } = useAssistant()
+  const navigate = useNavigate()
+
+  const openAssistant = () => {
+    if (!isAssistantEnabled()) return
+
+    const conversationId = makeConversationId()
+    onAssistantExecute({ value: '', conversationId })
+    navigate(`assistant/${conversationId}`)
+  }
+
+  return (
+    <AssistantProvider>
+      {children({
+        openAssistant
+      })}
+    </AssistantProvider>
+  )
+}
+
+const AssistantLinkWrapper = ({ children }) => {
+  return (
+    <AssistantProvider>
```

Comment:
it will be 2 AssistantProvider in the app, isn't it?

Reply:
It is possible to have two AssistantProvider now yes. Current code organization of cozy-search is less and less relevant for what we want to do with Twake theme. I think we will need to take times to refactor it like cozy-bar...

Reply:
I think we can just add in the documentation that the provider is needed, and don't add it in the exposed component